### PR TITLE
[CROSSDATA-465][testAT] Acceptance tests for INSERT INTO

### PR DESCRIPTION
**Pull request by hdominguez1989**
## Description

[CROSSDATA-465][testAT] Acceptance tests for INSERT INTO for ES, C* and MongoDB

### CODE REVIEW

**Comment by pianista215** [comment]
Nice job! :+1: :+1: :+1: 

May be would be good to put some Inserts with arrays, maps, maps that contains arrays, and arrays of maps like in the IT tests...

File: testsAT/src/test/resources/features/Cassandra/CassandraInsertInto.feature [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    When I execute 'SELECT count(*) FROM insertintotable2'
+    Then The spark result has to have '1' rows:
+      |_c0-long|
+      | 1     |
+
+  Scenario: [CROSSDATA-465] INSERT INTO notexitstable (ident,money,new,date,name) VALUES (0, 10.2 ,true, '1999-11-30 00:00:00', 'name_0')
+    When I execute 'INSERT INTO notexitstable (ident,money,new,date,name) VALUES (0, 10.2 ,true, '1999-11-30 00:00:00', 'name_0')'
+    Then an exception 'IS' thrown
+
+  Scenario: [CROSSDATA-465] INSERT INTO over TEMPORARY TABLE must return an exception
```
Comment (miguel0afd) [inline testsAT/src/test/resources/features/Cassandra/CassandraInsertInto.feature]: must **not** return, isn't it?

File: testsAT/src/test/resources/features/Cassandra/CassandraInsertInto.feature [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    When I execute 'CREATE TEMPORARY VIEW temptable AS SELECT ident FROM insertintotable3'
+    Then an exception 'IS NOT' thrown
+    When I execute 'INSERT INTO temptable(ident) VALUES (1)'
+    Then an exception 'IS' thrown
+
+  Scenario: [CROSSDATA-465] INSERT INTO insertintotable4 two rows in the same sentence
+    When I execute 'INSERT INTO insertintotable4 (ident,money,new,date,name) VALUES (0, 10.2 ,true, '1999-11-30 00:00:00', 'name_0'), (1, 11.2 ,true, '1999-11-30 00:00:00', 'name_1')'
+    Then an exception 'IS NOT' thrown
+    When I execute 'SELECT count(*) FROM insertintotable4'
+    Then The spark result has to have '1' rows:
```
Comment (miguel0afd) [inline testsAT/src/test/resources/features/Cassandra/CassandraInsertInto.feature]: have '2' rows

File: testsAT/src/test/resources/features/Elasticsearch/ElasticSearchInsertInto.feature [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    When I execute 'INSERT INTO temptable(ident) VALUES (1)'
+    Then an exception 'IS' thrown
+
+  Scenario: [CROSSDATA-465] INSERT INTO insertintotable4 two rows in the same sentence
+    When I execute 'CREATE EXTERNAL TABLE insertIntotableES4 (ident INT, name STRING, money DOUBLE, new BOOLEAN) USING com.stratio.crossdata.connector.elasticsearch OPTIONS (es.resource 'databasetest/insertIntotableES4', es.nodes '${ES_NODE}', es.port '${ES_PORT}', es.nativePort '${ES_NATIVE_PORT}', es.cluster '${ES_CLUSTER}')'
+    Then an exception 'IS NOT' thrown
+    When I execute 'INSERT INTO insertIntotableES4 (ident,money,new,name) VALUES (0, 10.2 ,true, 'name_0'), (1, 11.2 ,true, 'name_1')'
+    Then an exception 'IS NOT' thrown
+    When I execute 'SELECT count(*) FROM insertIntotableES4'
+    Then The spark result has to have '1' rows:
```
Comment (miguel0afd) [inline testsAT/src/test/resources/features/Elasticsearch/ElasticSearchInsertInto.feature]: Same as above

File: testsAT/src/test/resources/features/Mongo/MongoDBInsertInto.feature [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    When I execute 'INSERT INTO temptable(ident) VALUES (1)'
+    Then an exception 'IS' thrown
+
+  Scenario: [CROSSDATA-465] INSERT INTO insertIntotableMongo4 two rows in the same sentence
+    When I execute 'CREATE EXTERNAL TABLE insertIntotableMongo4 (ident INT, name STRING, money DOUBLE, new BOOLEAN, date TIMESTAMP) USING com.stratio.crossdata.connector.mongodb OPTIONS (host '${MONGO_HOST}:${MONGO_PORT}', database 'databasetest', collection 'insertIntotableMongo4')'
+    Then an exception 'IS NOT' thrown
+    When I execute 'INSERT INTO insertIntotableMongo4 (ident,money,new,date,name) VALUES (0, 10.2 ,true, '1999-11-30 00:00:00', 'name_0'), (1, 11.2 ,true, '1999-11-30 00:00:00', 'name_1')'
+    Then an exception 'IS NOT' thrown
+    When I execute 'SELECT count(*) FROM insertIntotableMongo4'
+    Then The spark result has to have '1' rows:
```
Comment (miguel0afd) [inline testsAT/src/test/resources/features/Mongo/MongoDBInsertInto.feature]: Same as above

**Comment by pianista215** [comment]
:+1:

**Pull request merged**